code/modules/clothing/gloves/color.dm

```diff
@@ -8,6 +8,11 @@
 	item_color="yellow"
 	burn_state = -1 //Won't burn in fires
 
+/obj/item/clothing/gloves/color/yellow/syndicate //Actually red, this is what you get when you just arbitrarily decide that one color is better than the rest!

```

Comment:
but then the insulated glove code is in two places, that's not very object oriented.

haha but seriously shitposting aside I left that comment because I'm hoping someone properly retypes insulated gloves down the line. I'd do it here but it'd turn the pull into a map edit, and there's been conflicts all over the place lately.